packages/init/src/index.ts

```diff
@@ -23,7 +23,8 @@ async function init() {
   const maybeRepositorySubdomain = findArgument(process.argv, "repository");
 
   Tracker.get().initialize(
-    process.env.NEXT_PUBLIC_SEGMENT_KEY || "JfTfmHaATChc4xueS7RcCBsixI71dJIJ",
+    process.env.PUBLIC_SM_INIT_SEGMENT_KEY ||
+      "JfTfmHaATChc4xueS7RcCBsixI71dJIJ",
```

Comment:
I think the default key here is not needed anymore. As SIroc holds the default value.

Reply:
it's more defensive to do it that way, if Siroc configuration change. "Bretelle and ceinture" 📦